index.html

```diff
@@ -1,55 +1,243 @@
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+             <h2 class="title">Multi-Post Stories <br>  Gain+Glory</h2>
+
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+             <h2 class="title">Multi-Post Stories <br>  Gain+Glory</h2>
+
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+             <h2 class="title">Multi-Post Stories <br>  Gain+Glory</h2>
+
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h2 class="title">Multi-Post Stories Gain+Glory</h2>
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="third-section">
+      <div class="container">
+        <img class="frameleft" src="./Images/frameleft.svg" alt="">
+        <img class="frameright" src="./Images/frameright.svg" alt="">
+        <h2>About Me</h2>
+        <p class="about-desc">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="box-button about-btn">Get My Resume</button>
+      </div>
+      <ul class="boxes">
+        <li class="about-box">
+          <div><img class="about-img" src="./Images/icon1.svg" alt="" /></div>
```

Comment:
- [ ] You have made a brilliant attempt here 😉 but has my colleague noted, please consider working on the `background-image` to ensure that they correctly loaded and positioned. Since these images have no relevance to the content other than for smooth aesthetics purpose and user experience, please consider removing them from normal flow in your `index.html` and import it using `background-image` properties and `values`.
- Check your design out:
![image](https://user-images.githubusercontent.com/99715640/212428980-1f6360e4-de0f-4522-889c-40bff0a8d406.png)

- Let's have a look at Figma design:
![image](https://user-images.githubusercontent.com/99715640/212429179-0a2739c1-0732-493e-88b9-d8eab255eb24.png)

- [ ] I suggest that you read more on `backgrounds` [here](https://www.w3schools.com/css/css_background.asp) on W3Schools.

Reply:
@AbayomiOlaoye  It is as good as impossible to download the background image in the about page section, that's why I can't style it through CSS. I have to use position absolute for each of the vectors in this case the vector image still tends to move, will it be acceptable to use the background image of the welcome page (The one in the first section)

Reply:
Oh, I'm sorry to hear about that 🙏. However there's a how. Let me know if you still have this issue via slack or just put a zoom call across. Let's see how to deal with this without taking your time for too long ☺️